### PR TITLE
[main] Update e2fsprogs to 1.46.4

### PR DESCRIPTION
**Pull request by christopherco**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update e2fsprogs to its latest version v1.46.4

###### Change Log  

- Update e2fsprogs spec to latest version
- Update cgmanifest to reflect new version
- Update toolchain files to build with new version

###### Does this affect the toolchain?  

**YES**

###### Test Methodology

local build

### CODE REVIEW

File: SPECS/e2fsprogs/e2fsprogs.spec [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Summary:        Contains the utilities for the ext2 file system
 Name:           e2fsprogs
-Version:        1.45.6
+Version:        1.46.4
 Release:        1%{?dist}
-License:        GPLv2 AND LGPLv2 AND BSD AND MIT
```
Comment (MaxBrodeurUrbas) [inline SPECS/e2fsprogs/e2fsprogs.spec:5]: Out of curiosity, how do you know to remove this?

File: toolkit/scripts/toolchain/container/toolchain-sha256sums [hunk 2]
```diff
@@ -24,7 +24,7 @@ a6bdd7d1b31266d11c4f4de6c1b748d4607ab0231af5188fc2533d0ae2438fec  diffutils-3.8.
 e72051e6a3e06b236d19eed56368117b745ca1e1a27bdc50fd51aa375bea6509  dmxproto-2.3.1.tar.bz2
 f9b55476def44fc7c459b2537d17dbc731e36ed5d416af7ca0b1e2e676f8aa04  dri2proto-2.8.tar.bz2
 01be49d70200518b9a6b297131f6cc71f4ea2de17436896af153226a774fc074  dri3proto-1.0.tar.bz2
-9bf7200d2737ed13f50a080af285c11529f91b088d84ecb71aae9fac58a8fbee  e2fsprogs-1.44.6.tar.gz
+7524520b291e901431ce59ea085955b601126de371bf3cfc0f5e4fad78684265  e2fsprogs-1.46.4.tar.gz
```
Comment (PawelWMS) [inline toolkit/scripts/toolchain/container/toolchain-sha256sums:27]: Weird that it's in this list but not build in the first stage of the toolchain but that's out of scope for this change, so LGTM.;)

**Review by PawelWMS — APPROVED** [review]

File: toolkit/scripts/toolchain/container/toolchain-sha256sums (hunk 2 above)
Comment (anphel31) [reply toolkit/scripts/toolchain/container/toolchain-sha256sums:27]: I think this is a good opportunity to clean this up - it doesn't look like it's needed. Can you remove this from both files?

**Review by anphel31 — APPROVED** [review]

File: toolkit/scripts/toolchain/container/toolchain-sha256sums (hunk 2 above)
Comment (christopherco) [reply toolkit/scripts/toolchain/container/toolchain-sha256sums]: Cleaned out e2fsprogs from the two files.

**Review by anphel31 — APPROVED** [review]

**Pull request merged**